opencti-platform/opencti-front/src/private/components/common/form/CreatorField.tsx

```diff
@@ -74,49 +89,73 @@ const CreatorField: FunctionComponent<CreatorFieldProps> = ({
         const NewCreators = (
           (data as CreatorFieldSearchQuery$data)?.members?.edges ?? []
         ).map((n) => ({
-          label: n?.node.name,
+          label: n?.node.name ?? t_i18n('Unknown'),
           value: n?.node.id,
+          extra: getExtraFromNode(n?.node),
         }));
-        const templateValues = [...creators, ...NewCreators];
+        const templateValues = [...creatorOptions, ...NewCreators];
         // Keep only the unique list of options
         const uniqTemplates = templateValues.filter((item, index) => {
           return (
             templateValues.findIndex((e) => e.value === item.value) === index
           );
         });
-        setCreators(uniqTemplates);
+        setCreatorOptions(uniqTemplates);
       });
   };
-
   return (
     <div style={{ width: '100%' }}>
       <Field
         component={AutocompleteField}
         name={name}
         textfieldprops={{
           variant: 'standard',
-          label: t_i18n(label),
-          helperText: helpertext,
+          label,
           onFocus: searchCreators,
         }}
+        disableClearable
         onChange={onChange}
         style={containerStyle}
         noOptionsText={t_i18n('No available options')}
-        options={creators}
-        isOptionEqualToValue={isOptionEqualToValue}
+        options={creatorOptions}
+        isOptionEqualToValue={(option: CreatorOption, selected: CreatorOption) => option.value === selected.value}
         onInputChange={searchCreators}
         renderOption={(
           props: React.HTMLAttributes<HTMLLIElement>,
-          option: { color: string; label: string },
+          option: CreatorOption,
         ) => (
-          <li {...props}>
-            <div className={classes.icon} style={{ color: option.color }}>
-              <ItemIcon type="user" />
-            </div>
-            <div className={classes.text}>{option.label}</div>
+          <li {...props} >
+            <Box
+              sx={{
+                paddingTop: 1,
+                color: option.color,
+              }}
+            >
+              <ItemIcon type="user"/>
+            </Box>
+            <Box
+              sx={{
+                flexGrow: 1,
+                marginLeft: 1,
+              }}
+            >
+              {option.label}
+            </Box>
+            {option.extra && (
+              <Box
+                sx={{
+                  flexGrow: 1,
+                  marginLeft: 1,
+                  textAlign: 'right',
+                  color: 'text.disabled',
+                }}
+              >
+                {option.extra}
+              </Box>
+            )}
           </li>
         )}
-        classes={{ clearIndicator: classes.autoCompleteIndicator }}
```

Comment:
this can be replaced by MUI's Autocomplete prop `disableClearable` (a global refactor could be done on this, I'll do it in another PR as it would ass too much noise here.)